bar_app/tests/reference/test_reference_model.py

```diff
@@ -0,0 +1,81 @@
+from django.test import TestCase
+from bar_app.models import Reference
+
+class ReferenceModelTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        Reference.objects.create(ref="REF001", name="name 1", description="description 1")
+        Reference.objects.create(ref="REF002", name="name 2")
+
+    def test_ref_label(self):
+        reference = Reference.objects.get(id=1)
+        field_label = reference._meta.get_field("ref").verbose_name
+        self.assertEqual(field_label, "ref")
+
+    def test_ref_max_length(self):
+        reference = Reference.objects.get(id=1)
+        max_length = reference._meta.get_field("ref").max_length
+        self.assertEqual(max_length, 100)
+
+    def test_ref_unique(self):
+        with self.assertRaises(Exception):
+            Reference.objects.create(ref="REF001", name="name", description="description")
```

Comment:
you could reuse the variable (class attributes) to make your tests more dynamic and less hardcoded

Reply:
also, this test is useless, because this is initially handled by DRF/django. You are making this test because you created useless validation methods.
You are basically testing the framework, and the people from django have done that for you a while back ;)